src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -76,7 +76,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
Again, I am not quite sure what this change does.

Reply:
This line of Java control what html the user can and cant use when writing messages. Using a Java library called Jsoup we can let users edit their text with html but only allowing certain basic tags to be used. I changed the line to Whitelist.basicWithImages to restrict the amount of html the user can use.